### ISSUE TITLE
How to connect to Azure DB when username contains @

### ISSUE THREAD

**Issue by fqzhang42** [pymssql/pymssql#911]
**Describe the bug**
fail to connect to Azure DB when the username contains @. I'm working on a company project, all username names are like id@company_domain. pymssql treats comany_domain  as the database server, results in connection failure.

**To Reproduce**
connect to Azure DB with an username contains @

**Expected behavior**
connect to Azure DB successfully

**Current Behavior**
pymssql thought the string after @ in the username is the server

**Comment by gordthompson** [comment]
If you are using SQLAlchemy then this is almost certainly a SQLAlchemy issue, not a pymssql issue.

**Comment by fqzhang42** [comment]
> If you are using SQLAlchemy then this is almost certainly a SQLAlchemy issue, not a pymssql issue.

No, I'm not. I just want to connect to Azure DB with Python. pymssql is the official one. But it doesn't work with username that contains @

**Comment by termim** [comment]
Please provide an example how you connect.

**Comment by fqzhang42** [comment]
> Please provide an example how you connect.
```python
import pymssql

connection = pymssql.connect(server='companyserver.database.windows.net', user='pid@domain.com', password='password', database='databasename')
```
